nasa-gibs/worldview
# Add Sentinel-3A and 3B Chlorophyll layers

## Description

Fixes #WV-2488 .

Added OLCI/Sentinel-3A and OLCI/Sentinel-3B Chlorophyll a layers.

## How To Test

1. Open with [these URL parameters](http://localhost:3000/?v=-177.80618840002722,-98.28510289906113,213.34590782217037,93.61307705065637&l=Reference_Labels_15m(hidden),Reference_Features_15m(hidden),Coastlines_15m,S3A_OLCI_Chlorophyll_a,S3B_OLCI_Chlorophyll_a,VIIRS_NOAA20_CorrectedReflectance_TrueColor(hidden),VIIRS_SNPP_CorrectedReflectance_TrueColor(hidden),MODIS_Aqua_CorrectedReflectance_TrueColor(hidden),MODIS_Terra_CorrectedReflectance_TrueColor&lg=true&t=2024-04-04-T17%3A17%3A34Z)
2. Check to see Chlorophyll a layers are loaded on map
3. Change date view recent dates of imagery (older dates have not yet been processed)
4. Go to the Layer Picker and make sure the layers show up under the Chlorophyll measurement,


## PR Submission Checklist

This is simply a reminder of what we are going to look for before merging your code.

- I have read the [CONTRIBUTING](https://github.com/nasa-gibs/worldview/blob/main/.github/CONTRIBUTING.md) doc
- I have added necessary documentation (if applicable)
- I have added tests that prove my fix is effective or that my feature works (if applicable)
- Any dependent changes have been merged and published in downstream modules (if applicable)

## Merging

Please use the `squash and merge` commit method unless each commit in your branch is vital to the commit history of main.

@nasa-gibs/worldview
